Mastermind.pm

```diff
@@ -172,24 +213,45 @@ sub run {
       my $mm_alt = $data_value->{alt};
 
       if( ($ref_allele eq $mm_ref && $alt_allele eq $mm_alt) || ($ref_allele_comp eq $mm_ref && $alt_allele_comp eq $mm_alt) ) { 
-        
+
+        # Only checks the genomic location - appends data for all transcripts   
+        if($self->{mutation_off}){
```

Comment:
The `if($self->{mutation_off})`  section should be before the `if ( ($ref_allele eq $mm_ref && $alt_allele eq $mm_alt) ...`.  If multiple mastermind records are present, only the one with the matching ref/alts will be reported even with the `mutation_off` option.  eg. `chr12   40340400        rs34637584      G       A`

Reply:
I'm not sure I understand what you mean, but`only the one with the matching ref/alts will be reported` this is true and it's the expected behavior.

Reply:
@dglemos : thanks for the explanation. I misunderstood the docs `all mutations/transcripts` as meaning all mutations at that position in the gene and not as all mutations at that position and with the same `ref/alt`.